content/docs/10.configuration-guide/url.md

```diff
@@ -42,19 +42,9 @@ micronaut:
 
 Make sure to check the [Micronaut HttpClient Configuration](https://docs.micronaut.io/latest/guide/configurationreference.html#io.micronaut.http.client.DefaultHttpClientConfiguration) for more information on how to configure the `DefaultHttpClientConfiguration` in your `config.yml` file.
 
-
-Some features within Kestra's embedded Code editor require network communication to work, such as the auto-update mechanism, querying and installing extensions, and telemetry. For these features to work properly in a proxy environment, you might need to adjust some firewall rules to allow communication with the Code editor. For example, Kestra's editor needs access to the `*.vscode-unpkg.net` domain when behind a proxy/firewall combination in order to load web extensions. If you are behind a firewall that needs to allow specific domains used by VS Code, make sure to allow communication to go through common hostnames such as `*.vscode-unpkg.net`. We recommend whitelisting the following URLs to make the VS Code editor work:
-- api.kestra.io
-- kestra-io.vscode-unpkg.net
-- proxzima.vscode-unpkg.net
-- az764295.vo.msecnd.net
-- marketplace.visualstudio.com.
-
 Another way to authenticate is to provide the `micronaut.http.client.proxy-authorization: Basic <base64-encoded username:password>` and
```

Comment:
Do review if this line is required or should be removed?

Reply:
Yup pretty sure this isn't required anymore. By using VSCode, we had the marketplace, internal updates and metrics data being sent. Now it's all in house, this isn't the case